vllm/model_executor/models/__init__.py

```diff
@@ -182,6 +185,10 @@ def register_model(model_arch: str, model_cls: Type[nn.Module]):
     def is_embedding_model(model_arch: str) -> bool:
         return model_arch in _EMBEDDING_MODELS
 
+    @staticmethod
+    def is_multimodal_model(model_arch: str) -> bool:
+        return model_arch in _MULTIMODAL_MODELS
```

Comment:
Let's dynamically check the model via `support_multimodal` (it can be loaded via `_try_load_model_cls`). That way, we won't have to hardcode `_MULTIMODAL_MODELS`.

Reply:
I don't think it's a big deal to hardcode `_MULTIMODAL_MODELS` since what we did was just moving models maps from one category to its own, and it's probably better to keep the same format here with embedding (and I guess encoder-decoder). WDYT?

Either way, I don't have strong opinion about doing it via `support_multimodal` if you think that's cleaner

Reply:
I think this may lead to issues in the future if we have models that belong to more than one category. It would be better to use `supports_multimodal`.

Reply:
That's a good point